cognite/neat/graph/stores/_base.py

```diff
@@ -64,11 +64,11 @@ def __init__(
 
         self.queries = Queries(self.graph, self.rules)
 
-    def add_rules(self, rules: InformationRules) -> None:
+    def add_rules(self, rules: InformationRules | AssetRules) -> None:
```

Comment:
Not sure if I agree with this one. Do we allow implementation rules in the Store?